.github/workflows/_build_artifacts.yml

```diff
@@ -171,7 +171,7 @@ jobs:
             platform: linux/arm/v7
             shortname: arm
         name:
-          - package: firezone-linux-client
+          - package: firezone-headless-client
             artifact: linux-client
```

Comment:
Not sure what this is controlling but might need changing?

Reply:
Yeah I don't know how the container registry is set up, not sure if it'll break anything customers are relying on. @jamilbk ?

Reply:
`artifact` is the binary name on GH releases, `image_name` is the Docker image.

The former we need to update docs for if it changes. The latter shouldn't change since `client` is enough naming I think:

`docker pull ghcr.io/firezone/client`